src/main/java/org/folio/service/orders/flows/update/open/OpenCompositeOrderFlowValidator.java

```diff
@@ -83,7 +94,7 @@ public Future<Void> checkLocationsAndPiecesConsistency(List<CompositePoLine> poL
     List<CompositePoLine> linesWithIdWithoutManualPieceReceived = poLines.stream().filter(
         compositePoLine -> StringUtils.isNotEmpty(compositePoLine.getId()) && Boolean.FALSE.equals(compositePoLine.getCheckinItems()))
       .collect(Collectors.toList());
-    List<String> lineIds = linesWithIdWithoutManualPieceReceived.stream().map(CompositePoLine::getId).collect(toList());
+    List<String> lineIds = linesWithIdWithoutManualPieceReceived.stream().map(CompositePoLine::getId).toList();
```

Comment:
Code smells about replacing Collectors.toList() with newer toList() need to be checked caredully. Each case requires careful inspectation because toList() collector produces unmodifieble list, which potentially could cause UnsupportedOperationException in case of modification attempt.

So please check and if we not modifying lineIds - it is ok to apply this refactoring.